web/src/settings_org.js

```diff
@@ -737,6 +738,33 @@ export function init_dropdown_widgets() {
         create_multiuse_invite_group_widget,
     );
     create_multiuse_invite_group_widget.setup();
+
+    const can_access_all_users_group_widget = new dropdown_widget.DropdownWidget({
+        widget_name: "realm_can_access_all_users_group",
+        get_options: () =>
+            user_groups.get_realm_user_groups_for_dropdown_list_widget(
+                "can_access_all_users_group",
+                "realm",
+            ),
+        $events_container: $("#settings_overlay_container #organization-permissions"),
+        item_click_callback(event, dropdown) {
+            dropdown.hide();
+            event.preventDefault();
+            event.stopPropagation();
+            settings_components.can_access_all_users_group_widget.render();
+            settings_components.save_discard_widget_status_handler($("#org-guest-settings"));
+        },
+        tippy_props: {
+            placement: "bottom-start",
+        },
+        default_id: page_params.realm_can_access_all_users_group,
+        unique_id_type: dropdown_widget.DATA_TYPES.NUMBER,
+        on_mount_callback(dropdown) {
+            $(dropdown.popper).css("min-width", "300px");
+        },
+    });
+    settings_components.set_can_access_all_users_group_widget(can_access_all_users_group_widget);
+    can_access_all_users_group_widget.setup();
```

Comment:
This isn't a task for this PR, but as a follow-up, we should work to make this a component where one can just pass in the setting name and it'll initialize everything; or maybe even a fancier version where one can declare it just via including the right block in an HTML template.

Reply:
Yeah, I was also thinking of doing something similar, would be needed especially when we would add more group-based settings.